### ISSUE TITLE
manual enrollment profile error

### ISSUE THREAD

**Issue by groob** [micromdm/micromdm#381]
when installing the enrollment profile manually this error shows up:

![screen_shot_2018-02-08_at_10 02 08_am](https://user-images.githubusercontent.com/1526945/35980410-7f6697d8-0c9f-11e8-8b16-f6dd2d5c85dd.png)

It has no effect on the actual enrollment, and users never see it during DEP enrollment anyway, but it would be great to get this fixed at some point.

**Comment by groob** [comment]
this error happens because the CN of the embedded root certificate is somehow not being read properly. 

Using the cert below in `<data>` for example gets rid of that problem... 
```
LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUNuRENDQVlRQ0NRQ0Q5QXY0N3hmZGtqQU5CZ2txaGtpRzl3MEJBUXNGQURBUU1RNHdEQVlEVlFRRERBVnoKWVc1MFlUQWVGdzB4T0RBeU1qWXhORE0wTWpGYUZ3MHhPVEF5TWpZeE5ETTBNakZhTUJBeERqQU1CZ05WQkFNTQpCWE5oYm5SaE1JSUJJakFOQmdrcWhraUc5dzBCQVFFRkFBT0NBUThBTUlJQkNnS0NBUUVBd3hSeVQweWxEWlZDCnNoOWZPK3BOVjZ6NGptQXdIQ2ZubVlPV2wzb0IzUlh2Q1pTT3JTNVRtMVNNN3JNdlduWlBCTGgzdnlnc3BiUzYKdXcyVVVzZ29JaW1nTVpjbndNOXJPK3FIUUMyYVZJWWlBRm1TTmMrUHpkZGRuZ0NlV3lGOGgyc0ZoZmtjaU5FKwpaaFJKR1JwNlV0bkFHVVdoNmxSV1pZSW9KZkgrSmJVQTdJaEFicXRBdWpXQWJPN294Z3VJdUJ2ZmY5akswQlc1Cm9kTGdhWjVQeUxVZFhJRnJpYW94aWZhVGROUXg5bXNOak55RVNVRXNDYWVZSWlSaGVCZnZDbEZUVjNtdTZHVmIKUmtObWJGNE5kMXB1eEUwNTFUeFF0S2NIbGE0YnAyd3dQMmVmNkx1dEVMVGprZ2trekowaUJSMjBwWHRNTVdmaAoyd0M4NE1zUU53SURBUUFCTUEwR0NTcUdTSWIzRFFFQkN3VUFBNElCQVFCYmh2YmlUbUpSTlpBTzhvVmlqNEdvCnN0QU1CMTZkUHV5L1FrcEtjV1dhRGtyc3ZwbG1NSzM1UE5pUS9MeGxFMGlHenQzcHBmN0VRN29ISmRaaXU3WkUKWmdMdVNIc2Z0UVVVRzVsZ0I0cCswUVdQbXJacUFhRi9HUk0xQU0yYW9QS2w1aG5wQXVaZFZpWUdTeHRPZ1l1aQpmcjR5R1RKK3U4MHVmUUZwSUFaMWRvakpELzNLSVNOaENVbUUvUlVPNzFUc0tvTG1mUEdWSE1ZZUtOY0FLVWxkCmdqVjFkYVZKRVN2NXJDTEk1YjNNa2NMTDNlWDdmVHBDV3ZqTnUyNjY4TDlwNU9JaTVrSDNRQjU4d2FYMlc3NEYKc2wyRzZ6akhwejBlWFIzUzI5a1JpTHhCZmhhTmJCQ1VrUTJ1TWZGcmxDY1hCUXdhTGs4c3ZhS056MFFwSkRTbAotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```

**Comment by groob** [comment]
Well this explains it:

https://github.com/micromdm/scep/blob/cced6af4d385f2129bc9d6f0969293dba542f668/cmd/scepserver/scepserver.go#L240

**Comment by jessepeterson** [comment]
This would also be solved with #278.
